runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractContextStreamView.java

```diff
@@ -57,6 +58,8 @@
     /** The base context, which is always preserved. */
     final T baseContext;
 
+    final Codec.Type codecType;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4705140)